ldap/src/integration-test/java/org/springframework/security/ldap/server/UnboundIdContainerTests.java

```diff
@@ -47,11 +51,12 @@ public void startLdapServer() throws Exception {
 		}
 	}
 
+
 	@Test
 	public void afterPropertiesSetWhenPortIsZeroThenRandomPortIsSelected() throws Exception {
-		UnboundIdContainer server = new UnboundIdContainer("dc=springframework,dc=org", null);
```

Comment:
We want to allow a null value for the LDIF file, since it is not required for the developer to provide one.